tox.ini

```diff
@@ -4,10 +4,11 @@ envlist = py34, py33, py32, py27, py27-configparser, pypy
 [testenv]
 passenv = HOME
 deps=
+  distribute
```

Comment:
This comes from an error I get on my Ubuntu 14.10 when running tox. It complains that 

ImportError: No module named 'setuptools'

I solved it by explicitly installing it in the virtualenv with this line. This error comes from the fact that I have no setuptools installed on my machine, probably.